core/arch/arm/kernel/thread_a64.S

```diff
@@ -379,31 +379,31 @@ END_FUNC thread_unwind_user_mode
 	.endm
 
 	.section .text.thread_vect_table
-	.align	11
+	.align	11, 0
```

Comment:
How could we make it more obvious that 0 is an invalid instruction? `#define INV_INSN 0` maybe?

Reply:
Commit subject: `s/arm32/arm64`

Reply:
I'll fix